data_structures/linked_list/doubly_linked_list.c

```diff
@@ -0,0 +1,296 @@
+/**
+ * @file
+ * @brief   Doubly linked list, simple code.
+ * @details
+ * A doubly linked list is a data structure with a sequence
+ * of components called nodes. Within that nodes there are
+ * three elements: a value recorded, a pointer to the next
+ * node, and a pointer to the previous node.
+ *
+ * In this implementation, the functions of creating the list,
+ * inserting by position, deleting by position, searching 
+ * for value, printing the list, and an example of how the
+ * list works were coded.
+ *
+ * See the algorithm explanation:
+ * [Doubly linked lists](https://en.wikipedia.org/wiki/Doubly_linked_list)
```

Comment:
ah, this seems to be the answer for https://github.com/TheAlgorithms/C/pull/673/files#r507199140
please see the suggestion above. This line can be removed

Reply:
Fixed up, thank you! :slightly_smiling_face: